main.js

```diff
@@ -0,0 +1,128 @@
+const core = require('@actions/core');
+const fs = require('fs');
+const path = require('path');
+const axios = require('axios').default;
+
+const srcPath = core.getInput('srcPath', { required: true });
+const dstPath = core.getInput('dstPath', { required: true })
+const dropboxToken = core.getInput('token', { required: true });
+core.setSecret(dropboxToken);
+
+// 150MB per upload limit on the Dropbox API
+const MAX_UPLOAD_BYTES = 157286400;
```

Comment:
Our builds have resource files and shared assets files of 450mb and 750 mb respectively does this mean they will fail to upload?

Reply:
Not sure how valid it is but this article mentions 350GB limit for API uploads https://help.dropbox.com/installs-integrations/sync-uploads/upload-limitations

Reply:
Honestly, was really hoping this would suffice for our current requirements so I wouldn't have to re-write a bunch of code :D

https://www.dropbox.com/developers/documentation/http/documentation#files-upload
```Do not use this to upload a file larger than 150 MB.```

There is an endpoint for larger files - https://www.dropbox.com/developers/documentation/http/documentation#files-upload_session-start  I'll investigate this...